### PR TITLE
Added Remove-RevitLocals function

### PR DESCRIPTION
**Pull request by BIMTim1**
Created the Remove-RevitLocals function to simplify the script and reduce the amount of repeated code. Full code notes have also been added to better clarify process of script.

### CODE REVIEW

File: CleanRevitTempFiles.ps1 [hunk 1]
```diff
@@ -5,50 +5,50 @@ $pathMain = "C:\Users\$user_file\AppData\Local\Autodesk\Revit"
...
+			#delete all files in the CollaborationCache folder
+			Get-ChildItem -Path "$MainPath\$fName\CollaborationCache" -Recurse | ForEach-Object {Remove-Item -Recurse -Path $_.FullName}
+			#delete all files in the Journals folder
+			Get-ChildItem -Path "$MainPath\$fName\Journals" -Recurse | ForEach-Object {Remove-Item -Recurse -Path $_.FullName}
+		}
+	}
+
+	Get-ChildItem -Path $TempFilePath -Recurse | ForEach-Object {Remove-Item -Recurse -Path $_.FullName -ErrorAction SilentlyContinue}
+
+	Get-ChildItem -Path "C:\`$Recycle.Bin" -Force | Remove-Item -Recurse -ErrorAction SilentlyContinue
```
Comment (ktnel) [inline CleanRevitTempFiles.ps1]: Add comments for these two actions.

File: CleanRevitTempFiles.ps1 [hunk 2]
```diff
@@ -5,50 +5,50 @@ $pathMain = "C:\Users\$user_file\AppData\Local\Autodesk\Revit"
...
+}
 #get Revit process in background
 $rev = "Revit"
 $rev_process = Get-Process -Name $rev -ErrorAction SilentlyContinue
 #check if Revit process is running
 if ($rev_process) {
 	$userInput = Read-Host "A Revit process is still running.`nType Y to close Revit. Type N to cancel tool."
 	if ("Y" -eq $userInput) {
```
Comment (ktnel) [inline CleanRevitTempFiles.ps1:32]: Currently, nothing happens if anything other than "Y" is typed. Do you want to display a cancel message (else statement)?

File: CleanRevitTempFiles.ps1 [hunk 3]
```diff
@@ -5,50 +5,50 @@ $pathMain = "C:\Users\$user_file\AppData\Local\Autodesk\Revit"
...
-			}
-		}
-
-		Get-ChildItem -Path $pathTemp -Recurse | ForEach-Object {Remove-Item -Recurse -Path $_.FullName -ErrorAction SilentlyContinue}
-
-		Get-ChildItem -Path "C:\`$Recycle.Bin" -Force | Remove-Item -Recurse -ErrorAction SilentlyContinue
-
-		Write-Host "All local Revit content has been removed from $user_file."
+		#pause for 10 seconds while Revit closes
+		Start-Sleep -Seconds 10
```
Comment (ktnel) [inline CleanRevitTempFiles.ps1]: How possible do you think it is for Revit to not fully close after 10 seconds? Can you come up with a While loop that would wait and confirm Revit is closed? Not sure if that would be necessary or not.

**Review by ktnel — CHANGES_REQUESTED** [review]
A couple of the comments should be considered. One is more of a question.

File: CleanRevitTempFiles.ps1 (hunk 2 above)
Comment (BIMTim1) [reply CleanRevitTempFiles.ps1:32]: I'll add an else {} statement to close if the user selects "N".

File: CleanRevitTempFiles.ps1 (hunk 3 above)
Comment (BIMTim1) [reply CleanRevitTempFiles.ps1]: There would be potential that it wouldn't close after 10 seconds, especially if the user is in the process of saving or syncing. I'll add a Do {} Until () statement to resolve this potential issue.

File: CleanRevitTempFiles.ps1 (hunk 1 above)
Comment (BIMTim1) [reply CleanRevitTempFiles.ps1]: Done

File: CleanRevitTempFiles.ps1 (hunk 2 above)
Comment (BIMTim1) [reply CleanRevitTempFiles.ps1:32]: I'll add an else {} statement to close if the user selects "N".

Comment (BIMTim1) [reply CleanRevitTempFiles.ps1:32]: `#delete all local temp files
	Get-ChildItem -Path $TempFilePath -Recurse | ForEach-Object {Remove-Item -Recurse -Path $_.FullName -ErrorAction SilentlyContinue}
	#empty recycle bin
	Get-ChildItem -Path "C:\`$Recycle.Bin" -Force | Remove-Item -Recurse -ErrorAction SilentlyContinue`

**Review by ktnel — APPROVED** [review]
All previous comments resolved.

**Comment by ktnel** [comment]
Merge complete

**Pull request merged**